cmake/std/atdm/spack-rhel6/environment.sh

```diff
@@ -136,8 +136,8 @@ export ATDM_CONFIG_BINUTILS_LIBS="${BINUTILS_ROOT}/lib/libbfd.so;${BINUTILS_ROOT
 # libbdf.a because the former works and the latter does not and TriBITS is set
 # up to only find static libs by default!
 unset BINUTILS_ROOT
-# NOTE: Above, you have to unset this or the configure dues in in the compiler
-# check.  What the heck?
+# NOTE: Above, you have to unset BINUTILS_ROOT or the CMake configure of
+# Trilinos dies in the compiler check.  What the heck?
```

Comment:
I'm objecting to "What the heck?".  While not obscene, it conveys an impression to users that we don't know what we're doing.  That impression may be not far from the truth ;-) but it's unnecessarily unsettling.

Reply:
@mhoemmen said:

> I'm objecting to "What the heck?".  While not obscene, it conveys an impression to users that we don't know what we're doing. That impression may be not far from the truth ;-) but it's unnecessarily unsettling.

This is scary that setting BINUTILS_ROOT breaks the compiler check for CMake.  We would have to track that down but this is scary.

Reply:
The NOTE already conveys that idea.

Look, people use this software to simulate Serious Things.  We need to sound serious here.  If something is broken, file an issue and put the issue number in the comment.

Reply:
Fixed in commit 15dce2b

Reply:
Argh, I realized that @bartlettroscoe had written this.  He is allowed to say "what the heck?!?" whenever he wants :-D